app/src/main/java/org/jellyfin/androidtv/preferences/enums/AppTheme.kt

```diff
@@ -0,0 +1,18 @@
+package org.jellyfin.androidtv.preferences.enums
+
+enum class AppTheme {
```

Comment:
Can we rename the themes (or the enums) to have consistent naming?

It doesn't make sense to me like this:
- Dark, Theme_Jellyfin
- Emerald, Theme_Jellyfin_Legacy
- Hotdog, Theme_Jellyfin_HotDogStand